### ISSUE TITLE
Does Catch integrate with XCTest?

### ISSUE THREAD

**Issue by audulus** [philsquared/Catch#371]
I noticed the following line in the [documentation](https://github.com/philsquared/Catch/blob/master/docs/why-catch.md)

> Implement test fixtures using Obj-C classes too (like OCUnit)

This is perhaps wishful thinking, but does Catch integrate with XCTest? I'd love to have cross-platform unit testing that also plays nice with Xcode.

**Comment by philsquared** [comment]
It doesn't, currently.
I was going to do a lot more integration work on the Obj-C side (including an iOS test runner) but with Swift gaining traction now I'm no sure it' worth it.

**Comment by MichaelSchmittQM** [comment]
What about https://github.com/philsquared/Catch/pull/454?

**Comment by usq** [comment]
#454 solves my xcode integration problem! @philsquared accepting the pull request would be awesome ;)

**Comment by fish2000** [comment]
@usq @philsquared strongly agree – I get a lot of mileage out of Catch for Objective-C++ unit testing but having this integration would be fantastic.

As for Swift: a lot of library code is getting retrofitted with `_Nonnull` qualifiers in order to be fully Swiftable; I for one would find it certifiably worth the effort (which admittedly is easy to say when the effort isn’t mine, erm) but yeah Catch and XCTest could be the chocolate and peanut butter of the impending Swift parifait, so to speak.

tl;dr: 👍🏽

**Comment by benthevining** [comment]
Any news on this? I'm open for any suggestions for how to run my unit tests when cross-compiling for iOS. I know CMake offers some XCTest integration, but all my unit tests are currently written for Catch, and I'd like to use Catch on all platforms, if possible. @horenmar

**Comment by TheStormN** [comment]
@philsquared any news on this one? This is becoming more pressing issue for cross platform projects. There are already some PRs and code suggestions. Could you at least merge some of them?
